### PR TITLE
Add entity that reads filled searchbox on new and old hosts pages

### PR DESCRIPTION
**Pull request by LadislavVasina1**
Add an entity `read_filled_searchbox` to the `host.py` (old hosts UI) and `all_hosts.py` (new hosts UI) so that it can be used in the automation.

Usage:

- Old UI

```python
with target_sat.ui_session() as session:
    session.host.search('some_search_string')
    read_searchbox_value = session.host.read_filled_searchbox()
    print(read_searchbox_value)
```

- New UI

```python
with target_sat.ui_session() as session:
    session.all_hosts.search('some_search_string')
    read_searchbox_value = session.all_hosts.read_filled_searchbox()
    print(read_searchbox_value)
```

### CODE REVIEW

**Review by vsedmik — APPROVED** [review]
Codewisely looks good to me, one question - shouldn't this be _always_ `True`?
```
read_searchbox_value == 'some_search_string'
```
Or what should I expect to change here and why?

**Comment by LadislavVasina1** [comment]
@vsedmik I don't know if I get your last comment, where do you specifically see usage of the `==` operator?

**Comment by vsedmik** [comment]
@LadislavVasina1 I just meant the example you provided in the PR description
```
with target_sat.ui_session() as session:
    session.all_hosts.search('some_search_string')
    read_searchbox_value = session.all_hosts.read_filled_searchbox()
    # here I would always expect read_searchbox_value == 'some_search_string'
    # so I just wondered why we need the read function
```

**Comment by LadislavVasina1** [comment]
@vsedmik I made this read function for vijay so he has an easier job in his current work, if that's what you are asking

File: airgun/entities/host.py [hunk 1]
```diff
@@ -84,6 +84,13 @@ def search(self, value):
         view = self.navigate_to(self, 'All')
         return view.search(value)
 
+    def read_filled_searchbox(self):
+        """Read filled searchbox"""
+        view = self.navigate_to(self, 'All')
+        self.browser.plugin.ensure_page_safe(timeout='5s')
+        view.wait_displayed()
+        return view.searchbox.read()
+
```
Comment (vijaysawant) [inline airgun/entities/host.py:93]: These changes are working absolutely fine, I tested it locally, and now running in CI as part of this robottelo PR https://github.com/SatelliteQE/robottelo/pull/16641
These changes are here in another airgun PR https://github.com/SatelliteQE/airgun/pull/1593

**Review by vijaysawant — APPROVED** [review]

**Pull request merged**